tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/BlockGridEditor/Document/BlockGridEditorInDocument.spec.ts

```diff
@@ -0,0 +1,276 @@
+
+    // Assert
+    await expect(page.locator('.umb-notifications__notifications > .alert-success', {hasText: "Document Type saved"})).toBeVisible();
+    // Checks if the new block list is in the group
+    await expect(page.locator('[data-element="group-' + documentGroupName + '"]', {hasText: blockGridName})).toBeVisible();
+    await expect(page.locator('[data-element="group-' + documentGroupName + '"]', {hasText: blockGridNameTwo})).toBeVisible();
+
+    // Clean
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementNameTwo);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockGridNameTwo);
+  });
+
+  test('Can change a block grid editor in a document to another block grid editor', async ({page, umbracoApi, umbracoUi}) => {
+    const blockGridNameTwo = 'BlockGridNameTwo';
+    const elementNameTwo = 'ElementNameTwo';
+    const elementAliasTwo = AliasHelper.toAlias(elementNameTwo);
+
+    await umbracoApi.documentTypes.ensureNameNotExists(elementName);
+    await umbracoApi.documentTypes.ensureNameNotExists(elementNameTwo);
+    await umbracoApi.dataTypes.ensureNameNotExists(blockGridNameTwo);
+
+    // Creates the first BlockGridEditor
+    const element = await umbracoApi.documentTypes.createDefaultElementType(elementName, elementAlias);
+    const BlockGridEditorOne = await createDefaultBlockGridEditorWithoutElement(umbracoApi, element, blockGridName);
+    // Creates the second BlockGridEditor
+    const elementTwo = await umbracoApi.documentTypes.createDefaultElementType(elementNameTwo, elementAliasTwo);
+    await createDefaultBlockGridEditorWithoutElement(umbracoApi, elementTwo, blockGridNameTwo);
+    // Creates Document with the first BlockGridEditor
+    await createDefaultDocumentTypeWithBlockGridEditor(umbracoApi, BlockGridEditorOne);
+
+    await umbracoUi.navigateToDocumentType(documentName);
+
+    // Switches from the first BlockGridBuilder to the second
+    await page.locator('[data-element="group-' + documentGroupName + '"] >> [data-element="property-' + blockGridAlias + '"] >> [title="Edit"]').click();
+    await umbracoUi.clickElement(umbracoUi.getButtonByKey('general_change'));
+    await umbracoUi.clickDataElementByElementName('datatype-Block Grid');
+    await umbracoUi.clickDataElementByElementName('datatypeconfig-' + blockGridNameTwo);
+    await umbracoUi.clickElement(umbracoUi.getButtonByLabelKey(ConstantHelper.buttons.submit));
+    await umbracoUi.clickElement(umbracoUi.getButtonByLabelKey(ConstantHelper.buttons.save));
+
+    // Assert
+    await expect(page.locator('.umb-notifications__notifications > .alert-success', {hasText: "Document Type saved"})).toBeVisible();
+    // We reload the page so we can see if the first BlockGridEditor was changed to the second BlockGridEditor
+    await page.reload();
+    // Needs to close tours when page has reloaded
+    await page.click('.umb-tour-step__close');
```

Comment:
I tried removing this, and it doesn't seem like it's needed, so not sure what this does 🤔